spicy/src/rt/parser.cc

```diff
@@ -94,9 +94,6 @@ bool detail::waitForInputOrEod(hilti::rt::ValueReference<hilti::rt::Stream>& dat
                                               data.get(), cur.size()));
         hilti::rt::detail::yield();
 
-        auto x = cur.end();
```

Comment:
I'm pretty sure this was on purpose originally because it triggers normalization of the stream iterator. I can't tell if we still need it. As long as all the btests pass, it should be fine to remove. The new stream implementation should also not need it anyways.

Reply:
The tests still pass.